lesson4/argirovatina_hometask.rb

```diff
@@ -11,7 +11,8 @@ def lazy?(key)
   end
 
   def respond_to_missing?(method_name, include_private = false)
-    self.has_key? method_name || super
+    key = get_correct_key(method_name)
```

Comment:
@argirovatina did you read my [second comment](https://github.com/nonkor/Betlab-ruby-courses/pull/35#discussion_r92581896)?

I understand why are you doing this. I've just pointed that it won't work properly for `#respond_to`. Try to call `LazyHash.new.respond_to?(:hash_key_which_not_exists)`